packages/block-library/src/freeform/modal.js

```diff
@@ -25,8 +25,7 @@ function ModalAuxiliaryActions( { onClick, isModalFullScreen } ) {
 
 	return (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
| Before | After     |
| :-------- | :------- |
|  <img width="915" alt="Classic-before" src="https://github.com/user-attachments/assets/d9f62399-4911-4141-b698-be9a82832cf4">  |  <img width="915" alt="classic-after" src="https://github.com/user-attachments/assets/4a691f40-9c3d-44c4-a7b7-8e8436cbb329">  |

Reply:
I noticed that the Modal close button hasn't had its size updated yet, so we'll need to resolve that first with the design team (#65102).

Could we perhaps revert just this instance (the fullscreen button) to the TODO state, so we can resolve those all together? The other instances in this file are ok.

Reply:
> I noticed that the Modal close button hasn't had its size updated yet, so we'll need to resolve that first with the design team (#65102).
> 
> Could we perhaps revert just this instance (the fullscreen button) to the TODO state, so we can resolve those all together? The other instances in this file are ok.

Sure @mirka, I would revert this change.

Reply:
Followed up in #65131